vendors/twtg/models/neon-tt_v2/model.yaml

```
@@ -12,6 +12,7 @@ product:
 deviceProfileId:
     - twtg_RFGroup1_RP2-1.0.4_classA
     - twtg_RFGroup2_RP2-1.0.4_classA
+    - twtg_RFGroup4_RP2-1.0.4_classA
```

This is not correct, right? The 1.0.4 is reffering the regional parameter version, which is:
```
regionalParameterVersion: RP2-1.0.1
```

The `twtg_RFGroup1_RP2-1.0.4_classA` and `twtg_RFGroup2_RP2-1.0.4_classA` were incorrect because I initially put the wrong regional parameter version. And I was not allowed to change the ID to be `twtg_RFGroup1_RP2-1.0.1_classA` and `twtg_RFGroup2_RP2-1.0.1_classA` for compatibility reasons, correct?

So I think it should be :
```suggestion
    - twtg_RFGroup4_RP2-1.0.1_classA
```

Or am I missing something here?

---

You're right. we had a mismatch between the values and the comment in the template. Finally, the id of the profile should contain the L2 and PHY  versions. 
the ID would be twtg_RFGroup4_1.0.4-RP2-1.0.1_classA.  
the other IDs of all device profiles 1.0.4 should be maintained later by us (for compatibility reasons).